feedbacks/tests.py

```diff
@@ -28,7 +29,7 @@ def test_invalid_form(self, value):
 @ddt
 class FeedbackListViewTest(DjangoTestCase):
     def setUp(self):
-        User.objects.create_user('user', password='test').save()
+        CustomUser.objects.create_user('user', password='test').save()
```

Comment:
Generally speaking it would be better to use the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper here instead of direct import